rpcd/playbooks/roles/logstash/templates/logstash

```diff
@@ -16,11 +16,12 @@ LS_OPTS="-w {{ logstash_workers|int }} {{ logstash_verbose|default() }}"
 
 # Arguments to pass to java
 {% if logstash_heap_size_mb is not defined %}
+{%   set logstash_heap_size_mb = 16384 %}
```

Comment:
Is this needed? It looks like this will get overridden by total//2 or total//3 ?